xml/geo_booth_i.xml

```diff
@@ -64,6 +64,73 @@
    parties.
   </para>
  </sect1>
+
+ <sect1 xml:id="sec.ha.geo.booth.ticket-types">
+  <title>Normal and Manual Tickets</title>
+  <para>
+   A ticket grants the right to run certain resources on a specific
+   cluster site. Two types of tickets are supported:</para>
+  <itemizedlist>
+   <listitem>
+    <para>Normal (or automatic) tickets are controlled by the &boothd; daemon.</para>
+   </listitem>
+   <listitem>
+    <para>Manual tickets are managed by the administrator only.</para>
+   </listitem>
+  </itemizedlist>
+
+  <para>Normal and manual tickets have a couple of interesting properties:</para>
+  <itemizedlist>
+   <listitem>
+    <formalpara>
+     <title>Normal and manual tickets can be defined together</title>
+     <para>It is possible to define manual and normal tickets in one Geo
+      cluster.</para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>Manual ticket management remains manual</title>
+     <para>The normal ticket management is not applied to manually controlled
+      tickets. Manual tickets do not have elections, automatic failover
+      procedures, and expirations.</para>
+    </formalpara>
+   </listitem>
+   <listitem>
+    <formalpara>
+     <title>Manual tickets are not moved</title>
+     <para>Tickets which were manually granted to a site, will remain there
+      until they are manually revoked. Even if a site becomes offline, the
```

Comment:
becomes offline -> goes offline? (or: Even if a site is no longer online, ...)

typo: tiket -> ticket

some services -> the services that depend on a ticket